core/application/commands.go

```diff
@@ -122,3 +131,40 @@ func (a *app) DeleteMicroVM(ctx context.Context, uid string) error {
 
 	return nil
 }
+
+func (a *app) addInstanceData(vm *models.MicroVM, logger *logrus.Entry) error {
+	instanceData := &cloudinit.Metadata{}
+
+	meta, ok := vm.Spec.Metadata["meta-data"]
+	if ok {
+		logger.Info("Instance metadata exists")
```

Comment:
Is there any extra logging data that could come out here? the MicroVM ID perhaps?

Reply:
Added.

Reply:
> Added.

where?